### DISCUSSION TITLE
how can i create a repository clone?

### DISCUSSION THREAD

**Discussion by JaazielTareas** [community/community#122831] [Accessibility]
### Select Topic Area

Question

### Body

how?

**Comment by Jaaziel-Polanco** [comment]
Click on code an open with gh desktop and select clone and open with vs-code

**Reply by ebndev** [reply]
Hi @Jaaziel-Polanco,

Please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
